src/js/expanderExpanded.js

```diff
@@ -54,7 +57,7 @@ function mdExpanderExpandedDirective($mdUtil, $animateCss) {
       })
       .start()
       .then(function () {
-        if (isHeightSet !== undefined) {
+        if (isHeightSet === true) {
```

Comment:
ya I agree with @JonathanTech.  If we want to do a type check lets wrap it in something like this?

```
if (typeof (variable) === "boolean") {

}
```